### PR TITLE
workflows: Disable builds for targets that aren't generally needed

### PR DESCRIPTION
**Pull request by hjpotter92**

### CODE REVIEW

**Review by thomshutt — APPROVED** [review]

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
           - amd64
         platform:
           - name: linux
-          - name: darwin
+          # - name: darwin
```
Comment (leszko) [inline .github/workflows/build.yaml:33]: I suggest removing this. Commented-out code is usually confusing.

**Review by leszko — APPROVED** [review]

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -26,11 +26,9 @@ jobs:
       fail-fast: false
       matrix:
         architecture:
-          - arm64
           - amd64
         platform:
```
Comment (pwilczynskiclearcode) [inline .github/workflows/build.yaml:30]: Could be simplified to:
```yaml
platform:
  - linux
```
we don't use `name:` key now

**Review by pwilczynskiclearcode — APPROVED** [review]
It might be worth to have manual builds (workflow_dispatch), nightly build or simply a separate workflow for `arm64` as it's becoming more popular on clouds (and less importantly `darwin`). Just saying.

**Comment by pwilczynskiclearcode** [comment]
@hjpotter92 Should we merge this change it?

**Pull request merged**